### DISCUSSION TITLE
Spacy ner v3 webtrf custom ner tags getting zero  entities

### DISCUSSION THREAD

**Discussion by badri-thinker** [explosion/spaCy#10179] [Help: Model Advice]
I trained a custom ner with web trf model using the ner_demo_update  (400 sentences with spans of text containing 2 custom entities) The new labels appear  in the eval but I have 0 entities.
PERSON        100.00    89.44    94.42
TITLE           0.00     0.00     0.00
ORG            98.21    95.51    96.84
GPE           100.00    97.80    98.89
CARDINAL      100.00    71.43    83.33
DATE           98.55    90.67    94.44
TIME          100.00    98.78    99.39
FAC           100.00    28.57    44.44
MONEY          98.63    83.72    90.57
PERCENT       100.00    90.91    95.24
CEONAME         0.00     0.00     0.00
LOC           100.00   100.00   100.00
ORDINAL       100.00   100.00   100.00
PRODUCT       100.00   100.00   100.00
WORK_OF_ART   100.00   100.00   100.00
LAW             0.00     0.00     0.00

**Comment by polm** [comment]
Please don't post the same thing in multiple threads. Since this is just a continuation of your other question let's keep discussion in #10064.
